specs/integration/search/2 - LoginFeature.js

```diff
@@ -0,0 +1,10 @@
+description: "Login with credentials.",
+
+scenario: [
+    LoginWidget.home(),
+    LoginWidget.open(),
```

Comment:
Could be renamed for better readability. As an external reader, I understand `home`, but not `open`.